lookup_plugin/s3_handler.py

```diff
@@ -265,13 +266,15 @@ def multiple_iteration_params(cluster_params, dirName, input_values):
             json_data = load_json_contents(newPath + "/dummy_generator.json")
             input_values["vdev"] = str(json_data['Vdev'])
             input_values["seqStart"] = str(json_data['SeqEnd'] + 1)
+            dbicount = str(json_data['TMinDbiFileForForceGC'])
```

Comment:
Why is the variable name TMinDbiFileForForceGC?
it should be dbi count right?

Reply:
dbi count is nothing but minimum count to perform Force garbage collection.

Reply:
if we see dbicount we may get confused , so mentioned with above key name.